src/gridster.component.ts

```diff
@@ -210,6 +210,32 @@ export class GridsterComponent implements OnInit, AfterContentInit, OnDestroy {
         this.ready.emit();
     }
 
+    adjustItemsHeightToContent(scrollableItemElementSelector: string = '.gridster-item-inner') {
+        this.gridster.items
+        // convert each item to object with information about content height and scroll height
+            .map((item: GridListItem) => {
+                const itemScrollableEl = item.$element.querySelector(scrollableItemElementSelector);
+                const itemScrollableElCoords = utils.getRelativeCoordinates(itemScrollableEl, item.$element);
+
+                console.log(itemScrollableEl.tagName);
```

Comment:
forgot to remove ?

Reply:
Yep, it's good that you keep an eye on my code. 👍